projects/demo/src/modules/components/textarea/index.html

```diff
@@ -1,170 +1,76 @@
-            </ng-template>
-            <ng-template
-                documentationPropertyMode="input"
-                documentationPropertyName="rows"
-                documentationPropertyType="number"
-                [documentationPropertyValues]="rowsVariants"
-                [(documentationPropertyValue)]="rows"
-            >
-                A number of visible rows in
-                <code>expandable</code>
-                mode
-            </ng-template>
-            <ng-template
-                documentationPropertyMode="input"
-                documentationPropertyName="tuiTextfieldIcon"
-                documentationPropertyType="PolymorpheusContent"
-                [documentationPropertyValues]="iconVariants"
-                [(documentationPropertyValue)]="icon"
-            >
-                Icon content. If content is a string, it is used as stringified svg or a name of an icon
+        <tui-doc-demo [control]="control">
+            <ng-template>
+                <tui-textfield
+                    [tuiTextfieldCleaner]="textfieldDoc.cleaner"
+                    [tuiTextfieldSize]="textfieldDoc.size"
+                >
+                    <label
+                        *ngIf="textfieldDoc.size !== 's'"
+                        tuiLabel
+                    >
+                        Label
+                    </label>
+                    <textarea
+                        placeholder="Placeholder"
+                        tuiTextarea
+                        [formControl]="control"
+                        [invalid]="controlDoc.invalid"
+                        [max]="max"
+                        [min]="min"
+                        [readOnly]="controlDoc.readonly"
+                        [tuiDisabled]="controlDoc.disabled"
+                    ></textarea>
+                </tui-textfield>
             </ng-template>
-        </tui-doc-documentation>
-        <inherited-documentation />
-        <tui-doc-documentation heading="Native <textarea> inside TuiTextarea">
-            <p>
-                Learn more how to set some attributes or listen to events on native textarea in this
-                <a
-                    fragment="native"
-                    tuiLink
-                    [routerLink]="routes.Textarea"
+        </tui-doc-demo>
+        <table tuiDocAPI>
+            <tbody>
+                <tr
+                    name="[min]"
+                    tuiDocAPIItem
+                    type="number"
+                    [(value)]="min"
                 >
-                    example
-                </a>
-            </p>
+                    Minimum number of rows in height
+                </tr>
+                <tr
+                    name="[max]"
+                    tuiDocAPIItem
+                    type="number"
+                    [(value)]="max"
+                >
+                    Maximum number of rows before scroll appears
+                </tr>
+            </tbody>
```

Comment:
Should we add `[limit]` directive for API page too ?
For possibility to explore it using different values.

Reply:
We would need to make it dynamic to support disabling with [limit]="0" otherwise it will always be on. While it might be helpful to do, I'd leave it until a feature request to not overcomplicate the code.